exercises/stack_03.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+// Stack.push implementation
+(function(){
+
+    function Stack(arr) {
+        this.arr = arr;
+        return;
+    }
+
+    Stack.prototype.forEach = function(callback) {
+        for (var i = 0; i < this.arr.length; i++) {
+            callback.call(this, this.arr[i], i, this.arr);
+        }
+
+        return;
+    }
+
+    Stack.prototype.push = function(item) {
+        this.arr.push(item);
+
+        return;
```

Comment:
empty return statements are equivalent to returning undefined, so aren't necessary for style